src/churn/churn.types.ts

```diff
@@ -8,4 +8,12 @@ export interface ChurnSignallingMessage {
   webrtcMessage ?:signals.Message;
   publicEndpoint ?:net.Endpoint;
   caesar ?:number;
+  obfuscator ?:ObfuscatorConfig;
+}
+
+export interface ObfuscatorConfig {
```

Comment:
I am in favor of phasing out terms relating to obfuscation in favor of something like transformers, as per our previous discussion on how obfuscation can have negative connotations.